source/stylesheets/base/_typography.scss

```diff
@@ -16,7 +16,6 @@ h4,
 h5,
 h6 {
   font-family: $heading-font-family;
-  font-size: modular-scale(1);
```

Comment:
I think this is a good change. Will watch for unintended consequences. I think we intenionally specify size on styled headings, so the risk is low.